.github/workflows/python-package-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: Python Package Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  nox-tests:
+    runs-on: ubuntu-latest
+    container: ghcr.io/gridtools/gridtools-base:${{ matrix.compiler }}
+    strategy:
+        matrix:
+            python-version: ["3.10"]
+            compiler: [gcc-10]
```

Comment:
I'm not sure, but it might make sense to also add `clang` to the matrix. What's your opinion, @havogt ?

Reply:
I don't think it's needed, it will not compile anything.

Reply:
would it be possible / better to just use the `gridtools-base:base` image?

Reply:
Probably will not work. We don't compile, but we still need to detect a proper compiler to configure the project (but this information is then not used).